packages/client/src/components/DisplayCard.component.tsx

```diff
@@ -0,0 +1,29 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import { WorkSuitIcon } from '@components/images/WorkSuitIcon';
+import { RestIcon } from '@components/images/RestIcon';
+
+type DisplayCardProps = {
+  item: {
+    title: string;
+    id: string;
+    hours: string;
+  };
+};
```

Comment:
item here seems unnecessary

```suggestion
type DisplayCardProps = {
    title: string;
    id: string;
    hours: string;
};
```